components/docs-chef-io/content/automate/ha_upgrade_introduction.md

```diff
@@ -79,4 +79,105 @@ We can also pass a flag in upgade command to avoid prompt for workspace upgrade.
    chef-automate upgrade run --airgap-bundle latest.aib --auto-approve --workspace-upgrade no
   ```
 
+{{< note >}}
+
+  AMI Upgrade is only for AWS deployment ,as in Onpremise Deployment all the resources are managed by customers its self.  
+{{< /note >}}
+
+## AMI Upgrade Setup For AWS Deployment
+
+### Requirements
+
+1. Two identical clusters located in same/different AWS regions.
+1. Amazon S3 access in both regions for Application backup.
+
+In the above approach, there will be two identical clusters
+
+- Primary Cluster (or Production Cluster)
+- Secondary Cluster (Having upgraded AMI)
+
+{{< note >}}
+
+  The AWS deployment should be configured with S3.  
+{{< /note >}}
+
+### Steps to set up the AMI Upgraded Cluster
+
+1. Deploy the Primary cluster following the deployment instructions by [clicking here](/automate/ha_aws_deploy_steps/#deployment).
+
+1. Deploy the Secondary cluster into a same/different data center/region using the same steps as the Primary cluster
+
+1. Do the backup configuration only when you have not provided the (backup information) configuration at the time of deployment. Refer backup section for [object storage](/automate/ha_backup_restore_aws_s3/).
+
+    {{< note >}}
+     During the deployment for the Primary and Secondary clusters, use the same S3 bucket name.
+    {{< /note >}}
+
+1. On Primary Cluster
+
+    - Take a backup of Primary cluster from bastion by running below command:
+
+    ```sh
+    chef-automate backup create --no-progress > /var/log/automate-backups.log
+    ```
+
+    - Create a bootstrap bundle; this bundle captures any local credentials or secrets that aren't persisted to the database. To create the bootstrap bundle, run the following command in one of the Automate nodes:
+
+    ```sh
+    chef-automate bootstrap bundle create bootstrap.abb
+    ```
+
+    - Copy `bootstrap.abb` to all Automate and Chef Infra frontend nodes in the Secondary cluster.
+
+
+1. On New AMI upgraded Cluster
```

Comment:
Either we can use `New Cluster with upgraded AMI` or Secondary Cluster(?)

Reply:
Done